bindings/pyroot/pythonizations/python/ROOT/pythonization/_tarray.py

```diff
@@ -8,6 +8,43 @@
 # For the list of contributors see $ROOTSYS/README/CREDITS.                    #
 ################################################################################
 
+'''
+/**
+\class TArray
+\brief \parblock \endparblock
+<div class="pyrootbox">
+\parblock
+## PyROOT
+
+When used from Python, the subclasses of TArray (TArrayC, TArrayS, TArrayI, TArrayL, TArrayF and TArrayD) benefit from the following extra features:
+
+- Their size can be obtained with `len`, which is equivalent to TArray::GetSize():
+
```

Comment:
Try removing the extra spaces before the code block. This way, it might be indented to the same level as the list items.
```suggestion
```

Reply:
Yes, it was intentional, this way there is more space between text and code, and visually it does not look bad to have the code indented leftmost, in my opinion.

Reply:
I actually like it the opposite way, but it's just visuals. :-)

Reply:
Just to compare, this is your option:

![image](https://user-images.githubusercontent.com/8089558/88274260-b2ecf800-ccdb-11ea-9444-c62265884854.png)

Reply:
Commited this option in the end, together with the fix for the extra box (use of `\htmlonly`).